src/test/java/org/mskcc/utils/ParserUtilTest.java

```diff
@@ -1,93 +1,93 @@
-package org.mskcc.utils;
-
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-
-import java.util.List;
-
-@SpringBootTest
-@RunWith(SpringRunner.class)
-public class ParserUtilTest {
```

Comment:
I checked out and uncommented - these succeed. Can we add them back in?

Reply:
I should not have committed the commented out class.  It is back.